src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_permute.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/ngraph.hpp"
+#include "./snippets_helpers.hpp"
+
+/* This file contains definitions of relatively simple functions (models) that will be used
+ * to test snippets-specific behavior. All the functions are expected to be direct descendants of
+ * SnippetsFunctionBase, so their constructors take only one (inputShapes) argument.
+ */
+
+namespace ov {
+namespace test {
+namespace snippets {
+/// Add separated from inputs by Sinh to WA CPU-specific disabling after inputs.
+/// Works because Sinh is not supported by tokenization yet.
+/// Tokenized simply by starting subgraph.
+//   in1       in2
+//   Sinh       Sinh
+//        Add
```

Comment:
Seems like we should align comment with the real graph :smile:

Reply:
Fair enough. fixed)